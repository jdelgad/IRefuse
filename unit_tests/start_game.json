{
  "action": "START",
  "players": 4
}