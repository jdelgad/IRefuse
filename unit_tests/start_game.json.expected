{"players": {"players": [{"cards": [], "tokens": 11, "number": 0}, {"cards": [], "tokens": 11, "number": 1}, {"cards": [], "tokens": 11, "number": 2}, {"cards": [], "tokens": 11, "number": 3}], "index": 0}, "cards": [29, 27, 32, 3, 30, 9, 15, 10, 34, 33, 11, 13, 14, 22, 26, 8, 35, 17, 19, 21, 4, 24, 12, 20]}